Lib/test/test_urllib.py

```diff
@@ -1503,6 +1518,48 @@ def test_url2pathname(self):
         self.assertEqual(fn('//localhost/foo/bar'), f'{sep}foo{sep}bar')
         self.assertEqual(fn('///foo/bar'), f'{sep}foo{sep}bar')
         self.assertEqual(fn('////foo/bar'), f'{sep}{sep}foo{sep}bar')
+        self.assertEqual(fn('data:blah'), 'data:blah')
+        self.assertEqual(fn('data://blah'), f'data:{sep}{sep}blah')
+
+    def test_url2pathname_require_scheme(self):
+        sep = os.path.sep
+        subtests = [
+            ('file:', ''),
+            ('FILE:', ''),
+            ('FiLe:', ''),
+            ('file:/', f'{sep}'),
```

Comment:
Just to be sure, but is `fILE:/` accepted or not? (namely is the scheme name case-sensitive?) (by the way, if `sep` is alone, you can just write `sep` instead of `f'{sep}'`)

Reply:
Scheme is case-insensitive in urllib, so `fILE:/` is accepted